### DISCUSSION TITLE
./autogen.sh fails on Ubuntu 16.04

### DISCUSSION THREAD

**Discussion by ghost** [sm00th/bitlbee-discord#224] [Q&A] [answered]
```
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -Im4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf --force
configure.ac:31: error: possibly undefined macro: AC_DISABLE_STATIC
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:32: error: possibly undefined macro: AC_PROG_LIBTOOL
autoreconf: /usr/bin/autoconf failed with exit status: 1

```

**Comment by sm00th** [comment]
do you have libtool installed?

**Comment by ghost** [comment]
That solved that issue. Now I'm seeing this, I can open a separate issue if you prefer.

```
checking for gcc option to accept ISO C99... none needed
./configure: line 12152: syntax error near unexpected token `BITLBEE,'
./configure: line 12152: `PKG_CHECK_MODULES(BITLBEE, bitlbee >= 3.5)'
```

```
BitlBee 3.5.1+20171005+master+27-g861de54-git
API version 030501
Configure args: --debug=0 --prefix=/usr --etcdir=/etc/bitlbee --events=libevent --otr=plugin --skype=0
```

I have bitlbee-dev installed.

**Comment by sm00th** [comment, marked as answer]
it now seems that pkg-config is missing. it is usually a good idea to install "build-essential" in ubuntu if you are trying to build something from source. that target inludes some basic stuff including libtool and pkg-config

**Comment by ghost** [comment]
build-essential was installed. I had to manually install some of the other things you mentioned. Would you accept a pull request that improved the readme with some of this information?

**Comment by sm00th** [comment]
> Would you accept a pull request that improved the readme with some of this information?

Sure thing, just make sure to mention that those package names are for debian/ubuntu-based distros.
